_docs/get_started/android.md

```diff
@@ -64,7 +64,7 @@ Unfortunately, Webview in Android 4.2 does not ship with a debugger – we will
 npm install -g weinre
 ```
 
-In order to make debugging easier, you will also want to tunnel to the local port that `weinre` runs on (the alternative is making sure your local computer and phone are on the same network and looking up your IP address, which is time-consuming and requires constant re-configuration). I use `ngrok` to do this. You can download it from [the ngrok website](`ngrok`).
+In order to make debugging easier, you will also want to tunnel to the local port that `weinre` runs on (the alternative is making sure your local computer and phone are on the same network and looking up your IP address, which is time-consuming and requires constant re-configuration). I use `ngrok` to do this. You can download it from [the ngrok website](https://ngrok.com/).
```

Comment:
@thisandagain Wasn't sure if I should touch that phrasing of the copy -- but totally agree. Will update this PR.